.changeset/famous-forks-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@uploadthing/mime-types": minor
```

Comment:
something's busted when we try and do minor here:

![CleanShot 2024-04-12 at 11 43 14](https://github.com/pingdotgg/uploadthing/assets/51714798/2cd1c37a-dd06-4b6a-b919-1143708d2403)

(shared should not get a major)

```suggestion
"@uploadthing/mime-types": patch
```